lib/pluggy_elixir/config.ex

```diff
@@ -0,0 +1,83 @@
+defmodule PluggyElixir.Config do
+  @moduledoc """
+  PluggyElixir client configurations
+
+  To use `PluggyElixir` you must add into your config file `config.exs` the
+  required config:
+
+      config :pluggy_elixir,
+        client_id: "your-app-client-id",
+        client_secret: "your-app-client-secret",
+
+
+  Allowed configuration:
+
+  - client_id(required): A client id provide by Pluggy
+  - client_secret(required): A client secret provide by Pluggy
+  - non_expiring_api_key: A boolean value to enable create a non expiring API
+  KEY. (default is `false`)
+  - sandbox: A boolean value to enable using sandbox `Connector`. (default is `false`)
+  - host: A string containing the API host. (default is `"api.pluggy.ai"`)
+
+  """
+
+  @default_host "api.pluggy.ai"
+
+  @doc false
```

Comment:
Will we have a doc here in the future or the idea is to have none at all?

Reply:
No because this function should not be used out of the lib scope.